tests/gui/pages/main.spec.js

```diff
@@ -260,8 +260,8 @@ describe('Browser: MainPage', function () {
     let DriveSelectionController
 
     const drivePaths = process.platform === 'win32'
-      ? [ 'E:\\', 'F:\\' ]
-      : [ '/dev/disk1', '/dev/disk2' ]
+      ? [ 'E:\\', 'F:\\', 'G:\\' ]
```

Comment:
These aren't device paths, but volume paths – should be `\\.\PhysicalDrive1` etc.

Reply:
Done